app/res/values/strings.xml

```diff
@@ -460,5 +460,5 @@
     <string name="entity_list_processing" cc:translatable="true">Processing cases: %s</string>
     <string name="entity_list_loading_cache" cc:translatable="true">Loading Cache…</string>
     <string name="entity_list_calculating" cc:translatable="true">Calculating cases %s</string>
-    <string name="entity_list_finalizing" cc:translatable="true">Finalizing list…</string>
+    <string name="entity_list_sorting" cc:translatable="true">Sorting cases…</string>
```

Comment:
Very nit-picky feedback - I don't know that 'sorting' here adds a lot of detail for users and might be a little tricky to translate, but "Finalizing" is also extremely technical, so I also wouldn't recommend switching back. I wonder if 'Finishing' might be a good, less jargon-y way to communicate.

Reply:
https://github.com/dimagi/commcare-android/pull/2928/commits/7530f944acd849ac4731affdb3bb128768fabf2a